src/public/javascripts/services/entrypoints.js

```diff
@@ -288,6 +288,9 @@ function registerEntrypoints() {
 
         searchNotesService.searchInSubtree(node.data.noteId);
     });
+
+    $('document').on('copy', utils.copySelectionToClipboard)
```

Comment:
what is the intention behind it? It seems to me like it is trying to change default "copy" behavior to this without formatting ... but a) it does not seem to work and b) default behavior should be copy with formatting IMHO ...

Reply:
You're right. I actually can't remember why I did this. This line clearly should not be there.

Reply:
OK, no problem, I removed it.